src/main/webapp/scripts/load.js

```diff
@@ -28,3 +28,18 @@ function loadObject(url) {
   const objPromise = fetch(url).then(promiseResponse => promiseResponse.json());
   return objPromise;
 }
+
+/**
+ * Fetches data from the keywords servlet, which includes salient terms from user input.
+ */
+function getSalientTerms() {
+  let keyTermPromise = loadObject('/keyword');
+  keyTermPromise.then((keyTerms) => {
+    console.log(keyTerms);
+
```

Comment:
[Nit] delete new line

Reply:
I think the new line before the comment block improves readability.